diff --git a/chrome/android/chrome_java_sources.gni b/chrome/android/chrome_java_sources.gni
index 425c67823fabe..9f81550bb0a31 100644
--- a/chrome/android/chrome_java_sources.gni
+++ b/chrome/android/chrome_java_sources.gni
@@ -813,6 +813,7 @@ chrome_java_sources = [
   "java/src/org/chromium/chrome/browser/media/AutoPipPermissionDialogView.java",
   "java/src/org/chromium/chrome/browser/media/DocumentPictureInPictureActivity.java",
   "java/src/org/chromium/chrome/browser/media/FullscreenVideoPictureInPictureController.java",
+  "java/src/org/chromium/chrome/browser/media/VideoAssistantController.java",
   "java/src/org/chromium/chrome/browser/media/MediaCaptureDevicesDispatcherAndroid.java",
   "java/src/org/chromium/chrome/browser/media/MediaCaptureNotificationServiceImpl.java",
   "java/src/org/chromium/chrome/browser/media/MediaLauncherActivity.java",
diff --git a/chrome/android/java/src/org/chromium/chrome/browser/tab/TabWebContentsDelegateAndroidImpl.java b/chrome/android/java/src/org/chromium/chrome/browser/tab/TabWebContentsDelegateAndroidImpl.java
index db1882e68a07a..50fc37d423e4c 100644
--- a/chrome/android/java/src/org/chromium/chrome/browser/tab/TabWebContentsDelegateAndroidImpl.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/tab/TabWebContentsDelegateAndroidImpl.java
@@ -35,5 +35,6 @@ import org.chromium.chrome.browser.bluetooth.BluetoothNotificationManager;
 import org.chromium.chrome.browser.gesturenav.NativePageBitmapCapturer;
 import org.chromium.chrome.browser.media.MediaCaptureNotificationServiceImpl;
+import org.chromium.chrome.browser.media.VideoAssistantController;
 import org.chromium.chrome.browser.policy.PolicyAuditor;
 import org.chromium.chrome.browser.policy.PolicyAuditorJni;
 import org.chromium.chrome.browser.serial.SerialNotificationManager;
